lib/resources/processes.rb

```diff
@@ -43,39 +43,37 @@ def to_s
 
     private
 
-    def ps_aux
+    def ps_axo
       os = inspec.os
 
       if os.linux?
-        command = 'ps auxZ'
+        command = 'ps axo label,pid,pcpu,pmem,vsz,rss,tty,stat,start,time,user:32,command'
```

Comment:
Just to add to it - Limiting the user field here since Usernames may only be upto 32 chars long. (Taken from man page - https://linux.die.net/man/8/useradd)
Can also use **args** instead of **command**.  Both can be used interchangeably.